pootle/apps/pootle_store/migrations/0027_unit_created_by_squashed_0055_fill_unit_source_data.py

```diff
@@ -0,0 +1,432 @@
+                ('source_hash', models.CharField(editable=False, max_length=32, null=True)),
+                ('source_length', models.SmallIntegerField(default=0, editable=False)),
+                ('source_wordcount', models.SmallIntegerField(default=0, editable=False)),
+            ],
+            options={
+                'abstract': False,
+                'db_table': 'pootle_store_unit_source',
+            },
+        ),
+        migrations.CreateModel(
+            name='UnitChange',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('changed_with', models.IntegerField(db_index=True)),
+                ('submitted_on', models.DateTimeField(db_index=True, null=True)),
+                ('commented_on', models.DateTimeField(db_index=True, null=True)),
+                ('reviewed_on', models.DateTimeField(db_index=True, null=True)),
+                ('commented_by', models.ForeignKey(null=True, on_delete=models.SET(pootle.core.user.get_system_user), related_name='units_commented', to=settings.AUTH_USER_MODEL)),
+                ('reviewed_by', models.ForeignKey(null=True, on_delete=models.SET(pootle.core.user.get_system_user), related_name='units_reviewed', to=settings.AUTH_USER_MODEL)),
+                ('submitted_by', models.ForeignKey(null=True, on_delete=models.SET(pootle.core.user.get_system_user), related_name='units_submitted', to=settings.AUTH_USER_MODEL)),
+                ('unit', models.OneToOneField(on_delete=django.db.models.deletion.CASCADE, related_name='change', to='pootle_store.Unit')),
+            ],
+            options={
+                'abstract': False,
+                'db_table': 'pootle_store_unit_change',
+            },
+        ),
+        migrations.RunPython(convert_unit_source_change),
+        migrations.RunPython(
+            remove_fields,
+        ),
+        RemoveFieldIfExists(
+            model_name='unit',
+            name='source_hash',
+        ),
+        RemoveFieldIfExists(
+            model_name='unit',
+            name='source_length',
+        ),
+        RemoveFieldIfExists(
+            model_name='unit',
+            name='source_wordcount',
+        ),
+        RemoveFieldIfExists(
+            model_name='unit',
+            name='commented_by',
+        ),
+        RemoveFieldIfExists(
+            model_name='unit',
+            name='commented_on',
+        ),
+        RemoveFieldIfExists(
+            model_name='unit',
+            name='reviewed_by',
+        ),
+        RemoveFieldIfExists(
+            model_name='unit',
+            name='reviewed_on',
+        ),
+        RemoveFieldIfExists(
+            model_name='unit',
+            name='submitted_by',
+        ),
+        RemoveFieldIfExists(
+            model_name='unit',
+            name='submitted_on',
+        ),
```

Comment:
All these `RemoveFieldIfExists` above look like a failsafe in case `remove_fields` that is run before those fails. Is it not possible to get rid of one of these, or to simplify this all in some way?